### ISSUE TITLE
Fields sow incorrectly in certain priority zones

### ISSUE THREAD

**Issue by CodeOptimist** [fluffy-mods/SpatialPriorities#1]
Tested with only `Mod Manager` and `Spatial Priorities 0.2.98` on `RimWorld 1.0.2059`
It seems pawns are sowing roses in the corn field.
If I click a pawn and right click the field to prioritize it, they performs the correct behavior and sow corn.
If I set a zone priority of either "Important" or "Critical" on the field it seems to fix it and the pawns sow the correct plant, but not with "Very Low", "Low", or "Normal".

(This attached log file is with all my mods enabled, not sure if that helps... I can attach one from a new colony with only _Spatial Priorities_ if necessary.)
[output_log.txt](https://github.com/FluffierThanThou/SpatialPriorities/files/2502498/output_log.txt)

(P.S. love your mods, huge fan!)

**Comment by flaviut** [comment]
I've also run into this bug, but when I went to take a look at source code, I couldn't find the Source submodule path.

**Comment by flaviut** [comment]
Oh and also: it's not always roses. It seems to plant whatever item is in the first growing zone that you created on your world.

**Comment by nathanmerrill** [comment]
I don't think this is limited to Normal and below:  I can reproduce this even if my plants are in an "Important" area

**Comment by CodeOptimist** [comment]
Oh really? Huh. I would like to experiment and figure out what's up with that, but I've a feeling someone else will beat me to it.

**Comment by nathanmerrill** [comment]
FYI: as a workaround, if you disable sowing on all fields except for one, that one field sows correctly.  When multiple fields are enabled, I'm not sure which one it chooses to sow. This means that, as a temporary workaround, you can enable them one at a time.

**Comment by CodeOptimist** [comment]
Ahh I wonder if it's looking at the wrong field's data. Just realized I do actually have some roses in potted plants on my map. I'll change them to something else and re-test.

**Comment by FluffierThanThou** [comment]
This is going to be a fun one to figure out... There's no reason I can think of why anything like this would happen. Either way, thanks a bunch for the feedback so far, I'll keep you guys up to date on my progress.

**Comment by CodeOptimist** [comment]
Just changed all my potted plants from Rose to Daylily and this did indeed change the corn field sowing to Daylily.
